deep-security-agent/README.md

```diff
@@ -1,89 +1,49 @@
-A collection of recipes to help deploy and manage the Deep Security agent.
+A collection of recipes to help deploy and manage the Trend Micro Deep Security Agent.
 
 ## Requirements
 
-All of the recipes in this cookbook require a working Deep Security infrastructure. The key component is the Deep Security manager. The agents (which these recipes help you manage) do the heavy lifting but the manager gives the marching orders. 
+All of the recipes in this cookbook require a working Trend Micro Deep Security infrastructure. The key component is the Trend Micro Deep Security Manager. The agents (which these recipes help you manage) do the heavy lifting but the manager gives the marching orders. 
```

Comment:
ok to use 'Trend Micro' here again, because it is referring to a different product (DSM)